fuel/app/classes/controller/welcome.php

```diff
@@ -40,7 +40,8 @@ public function action_index()
 	 * @return  Response
 	 */
 	public function action_hello()
-	{
+  {
```

Comment:
__[development][code_sniffer]__
* Tabs must be used to indent lines; spaces are not allowed
* Opening brace indented incorrectly; expected 1 spaces, found 2